src/dc_receive_imf.rs

```diff
@@ -1244,13 +1244,16 @@ async fn create_or_lookup_group(
         chat_id_blocked = create_blocked;
         recreate_member_list = true;
 
-        if create_protected == ProtectionStatus::Protected {
-            // set from_id=0 as it is not clear that the sender of this random group message
-            // actually really has enabled chat-protection at some point.
-            chat_id
-                .add_protection_msg(context, ProtectionStatus::Protected, false, 0)
-                .await?;
-        }
+        // once, we have protected-chats explained in UI, we can uncomment the following lines.
+        // ("verified groups" did not add a message anyway)
+        //
+        //if create_protected == ProtectionStatus::Protected {
+        // set from_id=0 as it is not clear that the sender of this random group message
+        // actually really has enabled chat-protection at some point.
+        //chat_id
+        //    .add_protection_msg(context, ProtectionStatus::Protected, false, 0)
+        //    .await?;
+        //}
```

Comment:
What about the idea to show this message but only if it's not at the start of a fresh verified chat? 
This way, a potential fully-protected-chat-enabed future release will still cause sensible messages in core47 releases.
Or am i missing something?

Reply:
well, if we're here, from the local view, we're _always_ at the start of a verified-group aka protected-chat. the group is just created a few lines above and is always empty. currently, we do not know if it is the first message in a chat or some subsequent message (and we missed the first ones or set up a new device or so).

explicit protected-enabled/disabled messages are shown, however, so, a core47 talking with a newer core will drop only the implicit enable-messages - as all cores up to now do with verified-group-enable. so, i'd say it is okay - unless i've missed sth :)